src/discretize.jl

```diff
@@ -16,17 +16,26 @@ function discretize(x::Vector, Flow::Union{ZigZag1d, Boomerang1d}, dt0)
     dt = dt0
     while k < length(x)-1
         while clock + dt <= τ
-            clock, ξ, θ = move_forward(dt, clock, ξ, θ, Flow)
+            if θ == 0.0
```

Comment:
What is this for?

Reply:
Just as a reference: the function `discretize` is implemented for the `ZigZag1d` and `Boomerang1d`. For the `Boomerang1d` with elliptic orbits not centred in 0, `move_forward` does not work when the particle is stuck in 0, since it would not stay in x,v = 0, 0.